back/src/bsda/validation/helpers.ts

```diff
@@ -23,10 +23,7 @@ import { safeInput } from "../../common/converter";
  * typés en string côté GraphQL mais en enum côté Zod ce qui nous oblige
  * à faire un casting de type.
  */
-export function graphQlInputToZodBsda(
-  input: BsdaInput,
-  isDraft = false
```

Comment:
Lors d'une mutation `update` on passait `isDraft = false` ce qui fait qu'on se retrouvait avec `updatedFields = ['isDraft']` puis l'erreur `Cannot read properties of undefined, reading sealed` au moment du check sur le verrouillage des champs. `isDraft` n'a en fait rien à voir dans cette fonction, j'ai déplacé la logique au niveau de la mutation `publish`.